src/types/tests/form-builder-api-test.ts

```diff
@@ -254,46 +274,6 @@ const options: Partial<FormBuilderOptions> = {
     return
   },
   layout: customLayoutClass,
+  controls: [control1, control2],
```

Comment:
@kevinchappell so would the custom control passed via options look something like this?